tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -2480,6 +2483,15 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     rinfo_.push_back({csinfo_.conv2d_grad_input,
                       mkl_op_registry::GetMklOpName(csinfo_.conv2d_grad_input),
                       CopyAttrsConv2D, AlwaysRewrite});
+    rinfo_.push_back({csinfo_.conv3d,
+                      mkl_op_registry::GetMklOpName(csinfo_.conv3d),
+                      CopyAttrsConv2D, AlwaysRewrite});  
```

Comment:
Maybe rename `CopyAttrsConv2D` to general CopyAttrsConv, and comment that Conv2D & Conv3D have the same attributes? Otherwise it looks like a typo, and I had to double check that it's correct.

Reply:
done, thanks